orders/views.py

```diff
@@ -1,3 +1,30 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+import jwt
+
+from django.http     import JsonResponse
+from django.views    import View
+
+from orders.models         import Cart
+from core.token_decorators import token_decorator
+
+class CartView(View):
+    @token_decorator
+    def patch(self, requset, cart_id):
+        try:
+            quantity = int(requset.GET.get('quantity'))
```

Comment:
보통 patch에 필요한 데이터를 body로 받습니다.

```suggestion
            quantity = int(requset.GET.get('quantity'))
```